charts/dgraph/templates/alpha/svc-headless.yaml

```diff
@@ -15,11 +15,12 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-  - name: grpc-alpha-int
-    port: 7080
-    targetPort: 7080
+    - name: grpc-alpha-int
+      port: 7080
+      targetPort: 7080
   selector:
     app: {{ template "dgraph.name" . }}
     chart: {{ template "dgraph.chart" . }}
     component: {{ .Values.alpha.name }}
     release: {{ .Release.Name }}
+  publishNotReadyAddresses: {{ .Values.alpha.service.publishNotReadyAddresses }}
```

Comment:
sorry late to the review:  
shouldn't this be corresponding to the `serviceHeadless` section of the Values and drop it from the service. otherwise it is confusing